easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-1.6.12-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '1.6.12'
```

Comment:
Basically; why is anything in this easyconfig different from just bumping the toolchain version in foss-2020a? Everything from dependencies to configopts is different here, and I'm afraid it seems to be for the worse

Reply:
As said above, meanwhile a newer version of EB is out. You can compile it as OpenMP and as MPI version.